src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -638,6 +623,16 @@ function showCurrent(state)
 
 })(document);
 </script><style>
+pre.sf-dump .sf-dump-compact,
+.sf-dump-str-collapse .sf-dump-str-collapse,
+.sf-dump-str-expand .sf-dump-str-expand {
+    display: none;
```

Comment:
There's one catch with this, it's that when JS is disabled, dumps are hidden.
We really need to hide them only when JS is enabled.

Reply:
Thanks for pointing this out! The PR has been updated.